docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -3110,6 +3120,58 @@ definitions:
         example: 'data.security.team'
 
 
+  Annotations:
+    type: object
+    description: >
+      # [Work in progress] Annotations of the Nakadi resource.
+
+      A Nakadi event-type or a subscription is considered as a Nakadi resource.
+      A Nakadi resource can have a set of annotations associated with it.
+      They follow the same rules of annotations in Kubernetes resources.
+        For keys:
+      > Annotations are key/value pairs. Valid annotation keys have two segments:
+      an optional prefix and name, separated by a slash (`/`). The name segment is
+      required and must be 63 characters or less, beginning and ending with an
+      alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`),
+      dots (`.`), and alphanumerics between. The prefix is optional. If specified,
+      the prefix must be a DNS subdomain: a series of DNS labels separated by dots
+      (`.`), not longer than 253 characters in total, followed by a slash (`/`).
+
+        For values:
+      Kubernetes don't have any restrictions for the values of annotations.
+      But Nakadi limits this to 1000 bytes at the moment, this can be relaxed in the future.
+
+    example:
+      nakadi-sql/query-input: "true"
```

Comment:
I would use more understandable example for the user who is not aware of Nakadi SQL. and this is how I thought it will look like e.g.: 
```
{
  "key1": "value1", 
  "key2": "value2"
}
```

Reply:
I'll change the example to a more generic one.